bcal.c

```diff
@@ -23,6 +23,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
+#include "dslib.h"
```

Comment:
Note that you'll need to change Makefile for this. I'ld also have to discuss with other collaborators on including dslib in Ubuntu, debian etc. However, this is a nice re-use.